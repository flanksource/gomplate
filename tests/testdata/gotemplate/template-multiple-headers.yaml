---
# yaml-language-server: $schema=https://raw.githubusercontent.com/fluxcd-community/flux2-schemas/main/helmrepository-source-v1beta2.json
# gotemplate: left-delim=<<< right-delim=>>>
name: Hello <<<.name>>>
