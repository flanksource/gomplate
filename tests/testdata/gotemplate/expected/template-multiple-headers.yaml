---
# yaml-language-server: $schema=https://raw.githubusercontent.com/fluxcd-community/flux2-schemas/main/helmrepository-source-v1beta2.json
name: Hello world